src/mentors.json

```diff
@@ -2203,5 +2203,34 @@
         "id": "3sanket3"
       }
     ]
+  },
+  {
+    "id": "ngohungphuc95@gmail.com",
+    "name": "Ngô Hùng Phúc",
+    "avatar": "https://media.licdn.com/dms/image/C5603AQG5w2tKUSPQ-Q/profile-displayphoto-shrink_200_200/0?e=1558569600&v=beta&t=Qr91t-JkFuT6nWjqZefzTIigTxyVQe_-8zu1NyBYkrk",
+    "title": "Senior Fullstack Software Engineer",
+    "description": "",
```

Comment:
Are you sure you don't want to provide any description? It will help people to decide or know more about you😄

Reply:
Agreed - I would add a description :)

Reply:
It's okay now guys